sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -673,6 +674,44 @@ def test_rst_example(gallery_conf):
     assert "rst-class:: sphx-glr-timing" in rst
 
 
+def test_recommendation_files(gallery_conf):
```

Comment:
The test is failing on the CI due to the `extras_require` dependency on numpy.

Reply:
You should add `numpy` to the testing requirements, too

Reply:
Ahh we can only add it some places... will push a fix